src/libsync/propagateremotedelete.cpp

```diff
@@ -81,7 +82,7 @@ void PropagateRemoteDelete::slotDeleteJobFinished()
 {
     propagator()->_activeJobList.removeOne(this);
 
-    Q_ASSERT(_job);
+    FASSERT(_job, "start() must be called");
```

Comment:
Would be a programming error, and would crash after anyway, I'd keep it a Q_ASSERT.